auto_update_tests.py

```diff
@@ -287,7 +287,7 @@ def update_ctor_eval_tests():
       print '..', t
       t = os.path.join('test', 'ctor-eval', t)
       ctors = open(t + '.ctors').read().strip()
-      cmd = WASM_CTOR_EVAL + [t, '-o', 'a.wast', '-S', '--ctors', ctors]
+      cmd = WASM_CTOR_EVAL + [t, '--disable-bulk-memory', '-o', 'a.wast', '-S', '--ctors', ctors]
```

Comment:
why is this specific flag needed here?

Reply:
to match it in check.py. It is needed in check.py because some tests like ctor-eval/imported2.wast have memory segments that begin with zeroes, so it is expected that memory packing will eliminate those zeroes. But memory packing only runs if bulk-memory is disabled. This will no longer be necessary once I change the default to be MVP, but I have a bunch of things in the pipeline that I want to get out before that happens.

Reply:
Ok. Please add a comment in both places then, that this should be removed.